templates/app/layouts/_default/baseof.html

```diff
@@ -4,6 +4,7 @@
   "environment" (or (getenv "HUGO_ENVIRONMENT") hugo.Environment)
 ) -}}
 
+{{/* TODO: How can we deal with this since .Params.content no longer exists? */}}
```

Comment:
Just change to `.Params.sys` as you've already done in `_default/singe.html` etc.
This only triggers a page render like in `_default/singe.html` to fill the scratch before the head is rendered.

Reply:
Done